### PR TITLE
#minor Fix single task execution node input generation

### PR DESCRIPTION
**Pull request by katrogan**
# TL;DR
This PR updates single task execution workflow generation such that the node containing the single task uses promises as inputs to pass through execution invocation inputs rather than statically bound values.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Discovered by @kumare3 

## Tracking Issue
https://github.com/lyft/flyte/issues/370

## Follow-up issue
_NA_

### CODE REVIEW

File: pkg/manager/impl/util/single_task_execution.go [hunk 1]
```diff
@@ -31,6 +31,8 @@ var defaultTimeout = ptypes.DurationProto(24 * time.Hour)
 
 const systemNamePrefix = ".flytegen.%s"
 
+const noInputNodeID = ""
```
Comment (wild-endeavor) [inline pkg/manager/impl/util/single_task_execution.go:34]: Propeller will know to fill this in with the start node?

Comment (katrogan) [reply pkg/manager/impl/util/single_task_execution.go:34]: yes, confirmed with @EngHabu offline

File: pkg/manager/impl/util/single_task_execution.go [hunk 2]
```diff
@@ -230,7 +105,7 @@ func CreateOrGetWorkflowModel(
 							Retries: &defaultRetryStrategy,
 							Timeout: defaultTimeout,
 						},
-						Inputs: generatedInputs,
+						Inputs: generateBindings(*task.Closure.CompiledTask.Template.Interface.Inputs, noInputNodeID),
```
Comment (kumare3) [inline pkg/manager/impl/util/single_task_execution.go:108]: shouldnt this be startNodeId?

File: pkg/manager/impl/util/single_task_execution.go (hunk 1 above)
Comment (kumare3) [reply pkg/manager/impl/util/single_task_execution.go:34]: ohh got it

**Pull request merged**